webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -18,10 +19,22 @@ import { DEFAULT_INTERVAL } from './constants';
 
 const interval = process.env.NOTIFICATIONS_POLLING || DEFAULT_INTERVAL;
 
-const getNotifications = url => get({ key: NOTIFICATIONS, url });
+const handleNotificationPollingError = (error, stopNotificationPolling) => {
+  if (error?.response?.status === 401) {
```

Comment:
Nitpick: `error` will always have a value
```suggestion
  if (error.response?.status === 401) {
```

Reply:
true, thanks :+1: